pages/common/go-doc.md

```diff
@@ -0,0 +1,12 @@
+# go doc
+
+> Tool for showing documentation for package or symbol.
+> More information: <https://golang.org/cmd/go/>.
+
+- Show documentation for {{Foo}} in the current package:
```

Comment:
I'd start with the most basic form of the command- "go doc". Explain what that does, then "go doc {{net/http}}", then "go doc {{net/http.Request}}". Also, I'd mention the `-src` and `-all` flag. They are important too.